public/js/render.js

```diff
@@ -20,8 +24,8 @@ var renderModule = function (error, apiData) {
 
     if (eventsArray.length === 0) {
       var noEvents = document.createElement('h1');
-      noEvents.innerHTML = 'NO UPCOMING EVENTS.';
-      noEvents.className = 'noEvents';
+      noEvents.innerHTML = 'No upcoming events.';
```

Comment:
it's going to be from the date that wqas chosen to the next 2 weeks so maybe "no upcoming events" is misleading but i guess it doesn't really matter what do you think?